.perltidyrc

```diff
@@ -0,0 +1,11 @@
+-l=400
```

Comment:
I think this should just continue using `~/.perltidyrc` because if the ustream one changes we’d have to update a zillion modules’ .perltidyrc instead of just `~/.perltidyrc`.

May not hurt to reference the prefered perltidyrc via a URL though

Reply:
Do you know of a place we keep that publicly?

Reply:
I don’t. Perhaps we can get permission to have it somewhere public and then I can add a blurb about it under a contributing guidelines section in any module I do. For now we should remove it I think.

Reply:
Fine.

Reply:
thanks